### DISCUSSION TITLE
Downscale core nodes

### DISCUSSION THREAD

**Discussion by alexa22013** [emqx/emqx-operator#628] [Q&A] [answered]
When I downscaled core nodes from 3 to 2 the repicant nodes were crashing 

replicant node logs showed this error
`
[emqx-replicant-6d5577f77d-lx4: 2023-02-26T19:08:47.518993+00:00 [error] line: 182, mfa: mria_lb:list_core_nodes/1, msg: mria_lb_core_discovery divergent cluster, node: 'emqx@10.237.115.78', previous_cores: [], returned_cores: ['emqx@emqx-core-0.emqx-headless.emqx.svc.cluster.local','emqx@emqx-core-1.emqx-headless.emqx.svc.cluster.local'], unknown_nodes: ['emqx@emqx-core-2.emqx-headless.emqx.svc.cluster.local']]
`
how to avoid this?

Also is there any practice on when to scale core node and when to scale replicants?

**Comment by Rory-Z** [comment, marked as answer]
Transfer to https://github.com/emqx/emqx-operator/issues/629

**Reply by Rory-Z** [reply]
[EMQX 5.0.19 is released](https://github.com/emqx/emqx/releases/tag/v5.0.19), this issue is fixed.
